### ISSUE TITLE
Is this project dead ?

### ISSUE THREAD

**Issue by sebma** [tomnomnom/gron#125]
Hi,
There has been no commit since Oct 29 2023 :

![image](https://github.com/user-attachments/assets/3c88b8e6-24ca-4150-925f-86c09f5a655b)

Is this project dead ?

**Comment by lafrenierejm** [comment]
Yes, this original repo is unmaintained. I have made some significant changes on [my fork](https://github.com/lafrenierejm/gron); I would be glad to accept feature requests or bug reports if you want to send them over there.

**Comment by tomnomnom** [comment]
I've been away for a while dealing with *things*, but I'm hoping to start picking things back up again shortly :)
